source/mongorestore/mongorestore-examples.txt

```diff
@@ -345,14 +345,37 @@ namespace:
        You cannot restore the ``system.buckets.weather.bson`` file
        by itself. Attempting to do so results in an error.
 
+Use mongorestore --oplogReplay and mongodump --oplog 
+----------------------------------------------------
+
+To back up a database to an :term:`oplog` ``oplog.bson`` file, use
```

Comment:
Can we delete lines 351 to 354, since they mention backing up the database to the oplog.bson file?

Reply:
Done